pennylane/__init__.py

```diff
@@ -82,13 +82,14 @@
 from pennylane.ops.functions import *
 from pennylane.optimize import *
 from pennylane.vqe import ExpvalCost, VQECost
-from pennylane.debugging import snapshots
+from pennylane.debugging.snapshots import snapshots
```

Comment:
```suggestion
from pennylane.debugging import snapshots
```

I think this can be reverted given `debugging/__init__.py`